flepimop/gempyor_pkg/src/gempyor/cli.py

```diff
@@ -20,12 +21,95 @@
 
 
 # add some basic commands to the CLI
-@cli.command(params=[config_files_argument] + list(config_file_options.values()))
-@pass_context
-def patch(ctx: Context = mock_context, **kwargs) -> None:
-    """Merge configuration files"""
+@cli.command(
+    params=[config_files_argument] + list(config_file_options.values()),
+    context_settings=dict(help_option_names=["-h", "--help"]),
+)
+@click.pass_context
+def patch(ctx: click.Context = mock_context, **kwargs) -> None:
+    """Merge configuration files
+
+    This command will merge multiple config files together by overriding the top level
```

Comment:
We should frame the example in the way we actually expect people to use the tool: `flepimop patch config1.yml config2.yml > confignew.yml`. Showing the output still useful so maybe do the piping, followed by a cat?

Reply:
Ah, good point, I've amended the last commit to reflect this.